kornia/losses/focal.py

```diff
@@ -60,8 +62,8 @@ def focal_loss(
             stacklevel=2,
         )
 
-    if not isinstance(input, torch.Tensor):
-        raise TypeError(f"Input type is not a torch.Tensor. Got {type(input)}")
+    if not isinstance(input, Tensor):
+        raise TypeError(f"Input type is not a Tensor. Got {type(input)}")
```

Comment:
`KORNIA_CHECK_SHAPE(input, ["B", "C", "*"])` which evnetually check for tensor too

Reply:
add also one too for the target similar to this one